POSConnector.js

```diff
@@ -254,27 +254,49 @@ var POSConnector = (function () {
      */
     connector.receiveMessage = function (message) {
         switch (message.name) {
-        case MessageName.BarcodeScanned:
-            handleBarcodeScannedMessage(message);
-            break;
-        case MessageName.GetLoginInformationCallback:
-            handleCallbackMessageWithParametersResultAndError(message);
-            break;
-        case MessageName.PayBasketCallback:
-            handleCallbackMessageWithParametersResultAndError(message);
-            break;
-        case MessageName.OpenURLCallback:
-            handleCallbackMessageWithParameterError(message);
-            break;
-        case MessageName.PrintDocumentCallback:
-            handleCallbackMessageWithParametersResultAndError(message);
-            break;
-        case MessageName.SendPOSConnectorObjectPathToPOSCallback:
-            handleCallbackMessageWithParameterError(message);
-            break;
+            case MessageName.BarcodeScanned:
+                handleBarcodeScannedMessage(message);
+                break;
+            case MessageName.GetLoginInformationCallback:
+                handleCallbackMessageWithParametersResultAndError(message);
+                break;
+            case MessageName.PayBasketCallback:
+                handleCallbackMessageWithParametersResultAndError(message);
+                break;
+            case MessageName.OpenURLCallback:
+                handleCallbackMessageWithParameterError(message);
+                break;
+            case MessageName.PrintDocumentCallback:
+                handleCallbackMessageWithParametersResultAndError(message);
+                break;
+            case MessageName.SendPOSConnectorObjectPathToPOSCallback:
+                handleCallbackMessageWithParameterError(message);
+                break;
         }
     };
 
+    /**
+     * @private
+     * @param {null | undefined | string | number} value
+     * @param {boolean} isNumber
+     * @return {null | string | number}
```

Comment:
Wrong return type. Passing in anything that is not number/string will return same value